docs/V7-Migration-Guide.md

```diff
@@ -356,20 +356,21 @@ Latest example can be found on [widgets.dojo.io/#widget/progress/overview](https
 ### radio
 
 #### Property changes
-##### Additional Mandatory Properties
-- foo: string
-	- this prop does x
-##### Changed properties
-- bar: string
-	- this prop replaced x
-	- this prop does foo bar baz
-	- more info
 ##### Removed properties
-- baz: string
-	- replaced by foo
-	- any additional info
+- label?: () => RenderResult
```

Comment:
this is just `RenderResult`

Reply:
Yep, will get this updated with the latest changes from this week.